backend/api/documents.py

```diff
@@ -0,0 +1,82 @@
+from typing import Any, Dict, Tuple, cast
+
+from flask import Blueprint, jsonify, request
+from flask_jwt_extended import get_jwt_identity, jwt_required
+
+import backend.services.documents_service as service
+from backend.mongodb_handler import mongo
+
+document_api = Blueprint('document_api', __name__)
+
+
+@document_api.route('/documents', methods=["POST"])
+@jwt_required()
+def create_document() -> Tuple[Any, int]:
+    body = request.get_json()
+    body, status_code = service.create_document(body, get_jwt_identity())
```

Comment:
There is a lot similar places in code that you return http codes from business logic layer. You mixed different levels of application. In my opinion there are two much better approaches:
1. Exceptions -- flask has a tooling for serializing python exceptions to json with needed http code. [Link](https://flask.palletsprojects.com/en/1.1.x/patterns/apierrors/)
2. You return True/False or return code (not http) from service layer: if everything is OK return True, else False. Or implement a set of return codes and propagate it from service layer to view layer with mapping to http codes
I would choose the first option because it is easier and better. Second option is also good, but not really good for web app actually, it is more for desktop apps